src/main/java/fi/aalto/cs/apluscourses/presentation/exercise/ExerciseViewModel.java

```diff
@@ -18,6 +18,38 @@ public String getPresentableName() {
     return APlusLocalizationUtil.getEnglishName(getModel().getName());
   }
 
+  /**
+   * Returns {@code true} if the exercise is submittable from the plugin, {@code false} otherwise.
+   */
+  public boolean isSubmittable() {
+    String presentableName = getPresentableName();
+    return presentableName.length() > "Assignment xx (".length()
+        && !"Assignment 1 (Piazza)".equals(presentableName);
```

Comment:
could you please extract both `String`s as constants (method/class level) and try coming out with some names that would give additional context (maybe like `A_BEGINNING_OF_THE_SUBMITTABLE_EXERCISE_NAME` or smth) :grin: